charts/redpanda/templates/_helpers.tpl

```diff
@@ -366,22 +366,22 @@ Generate configuration needed for rpk
 {{- end -}}
 
 {{- define "tunable" -}}
-{{- $tunable := dig "tunable" dict .Values.config }}
-{{- if (include "redpanda-atleast-22-3-0" . | fromJson).bool }}
-{{- toYaml $tunable | nindent 4 }}
-{{- else if (include "redpanda-atleast-22-2-0" . | fromJson).bool }}
-{{- $tunable = unset $tunable "log_segment_size_min" }}
-{{- $tunable = unset $tunable "log_segment_size_max" }}
-{{- $tunable = unset $tunable "kafka_batch_max_bytes" }}
-{{- toYaml $tunable | nindent 4 }}
-{{- else if (include "redpanda-atleast-22-1-1" . | fromJson).bool }}
-{{- $tunable = unset $tunable "log_segment_size_min" }}
-{{- $tunable = unset $tunable "log_segment_size_max" }}
-{{- $tunable = unset $tunable "kafka_batch_max_bytes" }}
-{{- $tunable = unset $tunable "topic_partitions_per_shard" }}
-{{- toYaml $tunable | nindent 4 }}
-{{- end }}
-{{- end }}
+{{- $tunable := dig "tunable" dict .Values.config -}}
```

Comment:
Why those `-` are necessary at the end?

Reply:
In _helpers.tpl, these are used to prevent the return from having any excess newlines. It's mostly just a good standard to follow.